src/pages/site/[uuid]/components/MockecData.ts

```diff
@@ -0,0 +1,102 @@
+import { MapPolygonCheckPanelItemProps } from "@/components/elements/MapPolygonPanel/MapPolygonCheckPanelItem";
+import { MapPolygonPanelItemProps } from "@/components/elements/MapPolygonPanel/MapPolygonPanelItem";
+
+export const uploadImageData = [
+  { id: "1", name: "Images5.png", status: "We are processing your image", isVerified: true },
+  { id: "2", name: "Images4.png", status: "We are processing your image", isVerified: true },
+  { id: "3", name: "Images3.png", status: "/home/solidgrids/Images3.png", isVerified: true },
+  { id: "4", name: "Images2.png", status: "/home/solidgrids/Images2.png", isVerified: false },
+  { id: "5", name: "Images1.png", status: "/home/solidgrids/Images1.png", isVerified: false }
+];
+
```

Comment:
Fix spelling of this file's name please (`MockedData.ts`)

Reply:
Done.